packages/charts/src/charts/donut/donut-total.component.tsx

```diff
@@ -1,43 +1,47 @@
 import React, { FC, memo } from 'react';
 
 import { Typography } from '@keen.io/ui-core';
-import { colors } from '@keen.io/colors';
 
 type Props = {
   children: string | number;
-} & Typography;
+  total: {
+    label: Typography;
+    value: Typography;
+  };
+};
 
-const DonutTotal: FC<Props> = memo(({ children, ...typography }) => {
-  const { fontColor, ...rest } = typography;
-  const { fontSize, fontFamily } = rest;
-  const totalLabelSize = 14;
+const DonutTotal: FC<Props> = memo(({ children, total }) => {
+  const { label, value } = total;
+  const { fontColor: labelColor, ...labelTypography } = label;
+  const { fontColor: valueColor, ...valueTypography } = value;
   return (
-    <>
-      <text
-        dx="0"
-        dy="0"
-        pointerEvents="none"
+    <text
+      y={`-${label.fontSize / 2}`}
+      pointerEvents="none"
+      style={{
+        textAnchor: 'middle',
+        dominantBaseline: 'middle',
+      }}
+    >
+      <tspan
```

Comment:
Is `<tspan>` a valid SVG element ? Why not used `<text>` here ?

Reply:
`tspan` is valid, here are the [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/tspan)

I use `<text>` to group label/value and display it in the middle, then add a vertical offset to substring in `<tspan>`